app/services/hungarian_comparator.rb

```diff
@@ -0,0 +1,26 @@
+class HungarianComparator
+  ALPHABET = Rails.configuration.x.hun_alphabet
+  CHARACTER_VALUES = {}
+  ALPHABET.each.with_index { |character, index| CHARACTER_VALUES[character] = index }
```

Comment:
This can be written with a one-liner too:
`CHARACTER_VALUES = ALPHABET.map.with_index {|char, i| [char, i] }.to_h`